app/controllers/runtime/organizations_controller.rb

```diff
@@ -339,11 +330,44 @@ def add_role(guid, role, user_id, username)
       [HTTP::CREATED, object_renderer.render_json(self.class, org, @opts)]
     end
 
+    def remove_role(guid, role, user_id)
+      response = remove_related(guid, "#{role}s".to_sym, user_id, Organization)
+
+      if config.get(:perm, :enabled)
```

Comment:
@chendrix @iredelmeier 
I think eventually we'll want the conditional calling of perm tested at the unit level to make sure we're calling perm when we want to with the expected stuff.

Since these attempt to make a network call, tests fail when the perm_client is invoked so it's clear that the feature flag is working.

Does that sound reasonable?